content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -0,0 +1,100 @@
+ 
+{{< note >}}
+The Open 3D Foundation and the broader Open 3D Engine community reserve the right to not publish any blog post that does not meet our standards. We also reserve the right to remove any currently published blog post.
+{{< /note >}}
+
+## What makes a good O3DE blog post?
+
+Good question! That's really up to the community alongside our established Code of Conduct and Community Tenets. Not sure? Join the O3DE Discord or one of the SIGs, and vet your idea with our members.
+
+Here's some basic Do's and Don'ts to help you focus your work and avoid rejection.
+
+### Do
+
+* Discuss O3DE! Go deep! Highly technical blog posts, in particular, establish community credibility and provide a resource for people looking for deeper knowledge beyond the tech docs.
+
+* Discuss algorithms, techniques, technologies, and tools that apply directly to O3DE work. Again, not only is your knowledge and experience useful to capture as a published topic, but it helps the larger industry assess the caliber of the people building O3DE.
+
+* O3DE strategy, **if** you are actively participating in a SIG around feature or product strategy, **and** have the approval of a SIG.
+
+* Interviews with, or profiles of, important and established community members. People put a human face on a technical product, and the world reliably likes to hear humans say what they have to say. In this case, it should be professional observations, knowledge, and commentary&mdash;and should be specifically related to O3DE. Personal opinions, as ever, should be largely technical.
+
+* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up." The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.
+
+* Include images and diagrams wherever appropriate.
+
+* Consider your audience, which is diverse across many sociocultural and industry axes. Choose your words with respect and care, and be prepared to take feedback sincerely and make changes. No one knows everything about the folks in the community and the industry, so don't let some initially careless wording caught in review bring you down. Make the changes (as appropriate) and keep delivering great content!
+
+### Don't
+
+* Get personal. Do not use the O3DE blog to call out any individuals in a negative way.
+
+* Get political. Politics pervade every part of life, and we all have different backgrounds, ideologies, and reactions to the way in which culture and society shapes itself. The O3DE blog isn't the place for it, unless you are running a community-approved political initiative and have the blessings of the Linux Foundation.
```

Comment:
```suggestion
* Get political. Politics pervade every part of life, and we all have different backgrounds, ideologies, and reactions to the way in which society shapes itself. The O3DE blog isn't the place for it&mdash;unless you are running a community-approved political initiative and have the blessings of the Linux Foundation.
```
Removed culture to avoid 'culture and society shape themselves'.  Em dashes are used elsewhere; it's the right choice here instead of a comma.